### PR TITLE
added new function

### PR DESCRIPTION
**Pull request by jonjonfell**
created new version of constructor test

### CODE REVIEW

File: js/tests/test.js [hunk 1]
```diff
@@ -15,6 +15,14 @@ describe('Account', function() {
 		});
 	});
 
+	describe('#constructor', function() {
```
Comment (navonod) [inline js/tests/test.js]: This is probably repetitive and also not DRY! Please fix.

File: js/tests/test.js [hunk 2]
```diff
@@ -15,6 +15,14 @@ describe('Account', function() {
...
+		it(`should initialise to a zero balance default & empty tx array- i made a change`, function() {
+			account = new Account("MyAccount");
+			expect(account.balance).to.equal(0);
+			assert(account.txs.length == 0);
```
Comment (navonod) [inline js/tests/test.js]: Yep, this all looks repeated

**Review by navonod — CHANGES_REQUESTED** [review]
Please look at my comments and make necessary correctIONS

**Comment by jonjonfell** [comment]
@navonod please see corrections and stuff

**Review by navonod — APPROVED** [review]
LGTM

**Pull request merged**